jetty-server/src/main/java/org/eclipse/jetty/server/HttpConnection.java

```diff
@@ -295,6 +295,14 @@ else if (filled < 0)
                 }
             }
         }
+        catch (Throwable x)
+        {
+            if (LOG.isDebugEnabled())
+                LOG.debug("{} caught exception {}", this, _channel.getState(), x);
+            BufferUtil.clear(_requestBuffer);
+            releaseRequestBuffer();
+            close();
+        }
```

Comment:
This is the only one of the new catch clauses  added that does not actually pass on the caught exception.
So perhaps add a comment saying why the close is enough... Actually, why are we not calling `getEndPoint().close(x);` ??

Reply:
Because there is no such method `EndPoint.close(Throwable)` in 9.4.x (it's only in 10).

Reply:
@lachlan-roberts @lorban can you fix in 10 to pass the throwable to close in your work on #6167

Reply:
@gregw will do.